### PR TITLE
Captcha Generator

### LINKED ISSUE #375: Custom captcha creator
To create custom captcha using html css and javascript.

### PR DESCRIPTION
**Pull request by sona-mathew-08**
# Description

This website creates a random captcha and validates the user input.
Credits: CodingNepal

Fixes #375 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions

**Test Configuration**:

- Operating system: Windows
- Version: 10
- Text-editors used: VS code

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
preview

### CODE REVIEW

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
image

Make it responsive for all screen sizes. Also, I think there is no need for main.js here so remove that file too

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-397--amazing-js-projects.netlify.app/captcha%20generator/

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
The box should be in the middle ig, rest is all perfect.

image

**Comment by VidhiBhatt01** [comment]
Enter the captcha box I mean

**Comment by sona-mathew-08** [comment]
preview5

**Review by DEEPIKA-SUK — APPROVED** [review]

File: color-guessing game/java/main.js [hunk 1]
```diff
@@ -1,105 +0,0 @@
-var allboxes= document.querySelectorAll('.box');
-var x=document.querySelector('#rgbspan');
-var color=generalRandomColor(6);
-var selectedColor=color[Math.floor(Math.random()*6)];
```
Comment (arpit456jain) [inline color-guessing game/java/main.js:4]: whats this @sona-mathew-08

**Comment by arpit456jain** [comment]
and @sona-mathew-08 delete all the preview images just put one with the name of preview.png

File: color-guessing game/java/main.js (hunk 1 above)
Comment (sona-mathew-08) [reply color-guessing game/java/main.js:4]: I didn't add that one. I don't know how it got added. But I had deleted the main.js file.

Comment (arpit456jain) [reply color-guessing game/java/main.js:4]: @sona-mathew-08 actually its the js file of another project please restore it and push the changes 
https://github.com/arpit456jain/Amazing-Js-Projects/blob/master/color-guessing%20game/java/main.js

**Comment by arpit456jain** [comment]
@sona-mathew-08 pull the latest changes ur branch is 2 commits behind

**Comment by sona-mathew-08** [comment]
> @sona-mathew-08 pull the latest changes ur branch is 2 commits behind

Is it ok now?

**Comment by arpit456jain** [comment]
> > @sona-mathew-08 pull the latest changes ur branch is 2 commits behind
> 
> Is it ok now?

push the changes to RP

**Pull request merged**